index.html

```diff
@@ -1,21 +1,289 @@
+                  <li><a href="#faq">FAQ</a></li>
+                  <li><a href="#contact">Contact</a></li>
+                </ul>
+              </nav>
+
+            </div>
+            <p>We Delivery to Your Doorstep</p>
+          </div>
+          <div class="menu_burger">
+            <svg viewBox="0 0 100 80" width="40" height="40">
+              <rect width="75" height="7"></rect>
+              <rect y="30" width="75" height="7"></rect>
+              <rect y="60" width="75" height="7"></rect>
+          </svg>
+          </div>
+        </div>
+      </header>
+      <main class="main">
+        <section class="hero">
+          <div class="hero-image"></div>
+          <section class="hero-content">
+            <div class="logo-icon-words">
+              <span class="logo-words"><p>ESTB</p></span>
+              <span class="logo-icon">
+                <img
+                  src="../responsive-grid-project/img/logo-icon.svg"
+                  alt=""
+                />
+              </span>
+              <span class="logo-words"><p>2023</p></span>
+            </div>
+            <h1>PLICKLES & CO</h1>
+            <h2>BRING NATURE INDOORS</h2>
+            <button class="btn shop-now">Shop Now</button>
+            <span class="chat-icon"></span>
+          </section>
+        </section>
+        <section class="products">
+          <div class="feature-and-image">
+            <div class="feature image-products">
+              <div class="feature-title-btn">
+                <h2>CACTI</h2>
+                <div class="border-bottom-h2"></div>
+                <button class="btn">Shop Collection</button>
+              </div>
+              <div class="feature-img" title="">
+                <img
+                  src="../responsive-grid-project/img/first-image.jpg"
+                  alt=""
+                />
+              </div>
+            </div>
+            <div class="feature image-products">
+              <div class="feature-title-btn">
+                <h2>PLANTS</h2>
+                <div class="border-bottom-h2"></div>
+                <button class="btn">Shop Collection</button>
+              </div>
+              <div class="feature-img" title="">
+                <img
+                  src="../responsive-grid-project/img/second-image.jpg"
+                  alt=""
+                />
+              </div>
+            </div>
+            <div class="feature image-products">
+              <div class="feature-title-btn">
+                <h2>SUCCULENTS</h2>
+                <div class="border-bottom-h2"></div>
+                <button class="btn">Shop Collection</button>
+              </div>
+              <div class="feature-img" title="">
+                <img
+                  src="../responsive-grid-project/img/third-image.jpg"
+                  alt=""
+                />
+              </div>
+            </div>
+          </div>
+          <div class="feature">
```

Comment:
This whole section will need to be created using CSS Grid. If you analyse how many columns there are in this layout, you can see the 'about us' text box spans half the screen, and the images next to it span a quarter each. How would you translate this using the `fr` unit? 

P.S. I recommend you rename this class as there are quite a few 'feature' classes floating around already.